src/abc_atlas_access/abc_atlas_cache/abc_project_cache.py

```diff
@@ -0,0 +1,495 @@
+from typing import Optional, Union, List
+from pathlib import Path
+import logging
+import pandas as pd
+from abc_atlas_access.abc_atlas_cache.cloud_cache import (
+    S3CloudCache,
+    LocalCache
+)
+
+
+class AbcProjectCacheVersionException(Exception):
+    pass
+
+
+def version_check(manifest_version: str,
```

Comment:
Is this an aborted idea?

Reply:
It's there now. This is to check that you aren't trying to load a manifest older than 2024. Since we turned on bucket versioning though, we attempt to update the old manifests with correct hashes with no real penalty if for some reason they don't work.

Reply:
The error message it raises does not use `cmax`.

We probably don't want to mention our confidence that this version is good until 2050. Maybe have a separate error message if `cmax` is violated (in case that ever becomes relevant).